pagegraph/src/graph_algos.rs

```diff
@@ -883,7 +884,6 @@ fn get_domain(host: &str) -> String {
         return host.to_string();
     }
     let source_hostname = host;
-    let source_domain = source_hostname.parse::<addr::DomainName>().expect("Source URL domain could not be parsed");
-    let source_domain = &source_hostname[source_hostname.len() - source_domain.root().to_str().len()..];
```

Comment:
I'm honestly not sure what this line is doing. Don't we just want the registrable domain? That's what we get from `.root()`.

Reply:
...yeah that looks like a better way to do it :sweat_smile: